### DISCUSSION TITLE
Emulator compatibility

### DISCUSSION THREAD

**Discussion by Savitar233** [bordplate/rac1-randomizer#37] [Improvement- and feature-suggestions]
Please, if it's possible, make the rando compatible with the rpcs3 emulator

**Comment by Myth197** [comment]
We got this request a lot in he discord and I wrote out an explanation of what needs to be done to get emulator comparability working.

1. The rando is currently built from RacMan, a tool used for making game modifications in real time on a console, you will have to fork and make modifications to it to make it compatible with rpcs3.
2. Make a PINE protocol script for rpcs3, then modify the rando to use that to communicate with the game.
3. Create a PS3 ISO patcher, then modify the rando to use that to generate modified game patches, or just straight up spew out a randomized ISO.

Currently no one has expressed interest in pursuing any of these options, but I'd be happy to see someone try.
